Unix/wsman/wsmanclient.c

```diff
@@ -42,21 +43,28 @@ struct _WsmanClient
     MI_Boolean sentResponse;
     WSBuf wsbuf;
     MI_Uint32 httpError;
+    MI_ConstString nameSpace;
+    MI_ConstString className;
+    MI_ConstString context;
+    MI_Uint32 endOfSequence;
 };
 
 static void PostResult(WsmanClient *self, const MI_Char *message, MI_Result result)
 {
-    PostResultMsg *errorMsg = PostResultMsg_New(0);
+    if ((MI_Boolean)Atomic_CompareAndSwap((ptrdiff_t*)&self->sentResponse, (ptrdiff_t)MI_FALSE, (ptrdiff_t)MI_TRUE) 
```

Comment:
You will need to make sentResponse into a ptrdiff_t otherwise you get memory corruption